src/components/ImageBanner/ImageBanner.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+
+import MediaCaption from 'Media/MediaCaption';
+import ImageElement from 'Image/ImageElement';
+
+type ImageBannerProps = {
+  alt: string;
+  caption?: string;
+  credit?: string;
+  className?: string;
+  sizes?: string;
+  src: string;
+  srcSet?: string;
+};
+
+const imageSizesDefault = '100vw';
+
+export const ImageBanner = ({
+  alt,
+  caption,
+  credit,
+  className,
+  sizes = imageSizesDefault,
+  src,
+  srcSet
+}: ImageBannerProps) => {
+  const classNames = cx('cc-image-banner', {
+    [`${className}`]: className
+  });
+
+  return (
+    <figure className={classNames}>
+      <div className="cc-image-banner__container" role="banner">
```

Comment:
Could this div and the previous figure be combined to simplify?

Reply:
If they can't be simplified .. perhaps ARIA role should be on figure anyway? Might make more sense?

Reply:
It looks like they could be just looking at it briefly so I've made this change now.

This is largely a copy and paste from Drupal 7 right now as it's WIP ... more to follow